Gemfile.lock

```diff
@@ -1038,6 +1072,7 @@ GEM
     zeitwerk (2.6.13)
 
 PLATFORMS
+  arm64-darwin-23
```

Comment:
Still shouldn't have this afaik.

Reply:
wait wtf

Reply:
oh both `arm64-darwin-22` and `arm64-darwin-23` were in there. weird. 💣